scripts/brp-remove-rpath

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# This script makes sure to remove any standard library search paths from rpath
+# or runpath tags as they're not only redundant, but will break overriding
+# search paths. Search paths of the tag that's not standard will be kept intact.
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
+	exit 0
+fi
+
+# make sure we have chrpath in our search path, otherwise wuzz out
+type chrpath || exit 0
```

Comment:
It'll only check whether binaries contains a RPATH or RUNPATH tag after finishing %install, so no need to know in advance.

As it pulls in a dependency on chrpath which might not be desired/available to have installed by default in build environment, it'll just wuzz out without checking nor fixing anything.